index.html

```diff
@@ -0,0 +1,279 @@
+            <span class="__label__list_container"></span>
+            <span class="__label__list_container"></span>
+            <span class="__label__list_container"></span></div> 
+      </header>
+    </nav>
+  <main>
+      <section> 
+        <div class="navigation">  
+            <h1 class="navigation__bar">I'm Alex Asciencio. Glad to see you!</h1>
+  
+              <p class="me">
+                I'm a software developer! I can help you build a product , feature
+                or website Look through some of my work and experience! If you like
+                what you see and have a project you need coded, don't hesitate to
+                contact me.
+              </p>
+        
+          
+            <h2 class="let__connect">LET'S CONNECT</h2>
+            <div class="social-media">
+              <ul>
+                <li><a href=""><img src="img/t_icon.png" alt="Twitter icon" /></a></li>
+                <li><a href=""><img src="img/Li_icon.png" alt="LinkedIn icon" /></a></li>
+                <li><a href=""><img src="img/m_icon.png" alt="Medium icon" /></a></li>
+                <li><a href=""><img src="img/_gth_icon.png" alt="Github icon" /></a></li>
+                <li><a href=""><img src="img/finger_c.png" alt="finger icon" /></a></li>
+            
+            </ul>
+           </div>
+        </div>
+         </section>
+         <section class="section__card">
+            <div class="navigation__work">
+              <img src="img/work/port1.png" class="portfolio_f_image" alt="" />
+              <div class="portfolio__img__content">
+              <h3 class="portfolio__tonic">Tonic</h3>
+              <div class="feature__container">
+                <div class="feature__container_side">
+                  <p class="category__portfolio">CANOPY</p>  
+                </div>
+                  <div class="canopy__design"></div>
+                  <div class="canopy__layout">
+                   <p class="canopy__backend">Back End Dev</p>
+                   
+                  </div>
+                  <div class="backend__design">
+                  <p class="category__port">2015</p>
+                  </div>
+              </div>
+              <p class="navigation__paragraph">
+                A daily selection of privately personalized reads; no accounts or
+                sign-ups required</p>
+              <div>
+                <div class="skills">
+                  <div class="skills__container">
+                    <p>HTML</p>
+                  </div>
+                  <div class="skills__container">
+                    <p>CSS</p>
+                  </div>
+                  <div class="skills__container">
+                    <p>JavaScript</p><br/>
+                  </div>
+              </div>
+              <div class="Portfolio__project__layout">
+                <div class="Portfolio__project__design">
+                  <div class="Portfolio__project__content">See Project </div>
```

Comment:
Hi, @AsciencioAlex!

As suggested by the previous reviewer, I noticed that you used a div tag for your see project button. However, the project requirements say that the button (e.g., "See project") should be a link a or button tag. This is important for semantic and accessibility reasons, as it helps to identify the purpose and functionality of your element and make it more accessible for keyboard and screen reader users.

To fix this issue, you should replace the div tag with a link a or button tag and add the appropriate attributes and styles. For example:

`<div class="see-project">See project</div> // Replace this with <a class="see-project" href="#">See project</a> or <button class="see-project">See project</button>`

Please make the change for all your project buttons

Please make this change and resubmit your project. Thank you! 😊